crawl-ref/source/timed-effects.cc

```diff
@@ -1258,3 +1259,104 @@ int speed_to_duration(int speed)
 
     return div_rand_round(100, speed);
 }
+
+static int& _zot_clock_for(branch_type br)
+{
+    static const string CLOCK_KEY_ROOT = "ZOT_CLOCK_FOR_";
+    const string branch_name = branches[br].abbrevname;
+    return you.props[CLOCK_KEY_ROOT + branch_name].get_int();
```

Comment:
I considered that, but it seemed more complex to me. Why do you think it's more robust?

Reply:
because it would make all zot clock data easily accessible and manipulable in a single data structure? For example easy to iterate through, etc. Also easier to rename things if it ever came to that, keeps the player prop namespace less polluted. There's no cost to doing it as its own hash table that I'm aware of.

Reply:
Iterating through it seems nice. I can do that.